src/fuzzing/DeserializeAddrMan/fuzz.cpp

```diff
@@ -11,3 +14,20 @@ int main (int argc, char *argv[]) {
         return -1;
     }
 }
+
+#endif
+
+#ifdef FUZZ_WITH_LIBFUZZER
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  CAddrMan addrman
+  CDataStream ds(Data, Data+Size, 0, 0); // TODO: is this right for type and version? 
+	try {
+		ds >> addrman;
+	} catch (const std::exception &e) {
+		return 0;
```

Comment:
```suggestion
    try {
        ds >> addrman;
    } catch (const std::exception &e) {
        return 0;
```
(replacing tabs with spaces)